tasks/configure.yml

```diff
@@ -44,6 +44,25 @@
   tags:
     - rstudio-workbench-configure-load-balancer
 
+- name: Create a directory for workbench in EFS
```

Comment:
Assuming the permissions and owner is the same across all directories, you can create all of them in a single task. 

https://docs.ansible.com/ansible/latest/collections/ansible/builtin/file_module.html#parameter-state

>If `directory`, all intermediate subdirectories will be created if they do not exist. Since Ansible 1.7 they will be created with the supplied permissions.